packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -363,18 +515,37 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
             id,
             minWidth,
             isOpen,
-            focusedIndex,
+            focusedIndex: undefined,
             enableVirtualization,
             filteredOptions,
             getMenuItem,
+            multiple,
           }}
         >
           <ComboBoxInputContainer disabled={inputProps?.disabled} {...rest}>
-            <ComboBoxInput
-              value={inputValue}
-              {...inputProps}
-              onChange={handleOnInput}
-            />
+            <>
+              <ComboBoxInput
+                style={{
+                  paddingLeft: tagContainerWidth + 18,
+                  minHeight: tagContainerHeight,
+                }}
+                value={inputValue}
+                {...inputProps}
+                onChange={handleOnInput}
+              />
+              {isMultipleEnabled(selectedIndex, multiple) && (
+                <MultipleSelectedContainer
```

Comment:
Have you researched the accessibility concerns that come with this markup? The multi-select implementation was already inaccessible (https://github.com/iTwin/iTwinUI-react/issues/794) and combobox, being more complex, requires even more thought to make accessible.

Reply:
Some time ago we discussed adding accessibility fixes in other PR.